doc/discretization.rst

```diff
@@ -0,0 +1,282 @@
+================
+
+To account for the discontinuities at element faces, DG employs numerical fluxes, which are
+enforced to be singled-valued, but are functions of both $\pm$ states:
+
+$$
+\mathbf{h}_e(\mathbf{Q}_h^+, \mathbf{Q}^-_h; \mathbf{n}) &\approx \mathbf{F}^I(\mathbf{Q}_h)
+\cdot\mathbf{n}, \\
+\mathbf{h}_v(\mathbf{Q}_h^+, \mathbf{\Sigma}_h^+, \mathbf{Q}_h^-, \mathbf{\Sigma}_h^-;
+\mathbf{n}) &\approx \mathbf{F}^V(\mathbf{Q}_h, \mathbf{\Sigma}_h)\cdot\mathbf{n}\\
+\mathbf{H}_s(\mathbf{Q}^+_h, \mathbf{Q}_h^-; \mathbf{n}) &\approx \mathbf{Q}_h\mathbf{n}.
+$$
+
+Choices of numerical fluxes corresponding to BR1
+------------------------------------------------
+
+* Take $\mathbf{h}_e(\mathbf{Q}_h^+, \mathbf{Q}^-_h; \mathbf{n})$ to be one of:
+
+  * Local Lax-Friedrichs (LLF), Roe, Engquist-Osher, HLLC (there are many more!). If you're
+    feeling especially brave, you can even use the average of the inviscid flux (centered
+    flux).
+
+  * Mirgecom uses LLF at the moment:
+    $$
+    \mathbf{h}_{e}(\mathbf{Q}_h^+, \mathbf{Q}^-_h; \mathbf{n}) = \frac{1}{2}\left(
+    \mathbf{F}^{I}(\mathbf{Q}_h^+)+\mathbf{F}^{I}(\mathbf{Q}_h^-)\right) - \frac{\lambda}
+    {2}\left(\mathbf{Q}_h^+ - \mathbf{Q}_h^-\right)\mathbf{n},
+    $$
+    where $\lambda$ is the characteristic max wave-speed of the fluid. Numerical fluxes
+    which penalize the ''jump'' of the state $\left(\mathbf{Q}_h^+ - \mathbf{Q}_h^-\right)
+    \mathbf{n}$ act as an additional source of dissipation, which has a stabilizing effect
+    on the numerics.
+
+* Take $\mathbf{h}_v(\mathbf{Q}_h^+, \mathbf{\Sigma}_h^+, \mathbf{Q}_h^-,
+  \mathbf{\Sigma}_h^-; \mathbf{n})$ to be defined in a ''centered'' (average) way:
+  $$
+  \mathbf{h}_v(\mathbf{Q}_h^+, \mathbf{\Sigma}_h^+, \mathbf{Q}_h^-, \mathbf{\Sigma}_h^-;
+  \mathbf{n}) = \frac{1}{2}\left(\mathbf{F}^V(\mathbf{Q}_h^+, \mathbf{\Sigma}_h^+) +
+  \mathbf{F}^V(\mathbf{Q}_h^-, \mathbf{\Sigma}_h^-)\right)\cdot\mathbf{n}
+  $$
+
+* And similarly for the gradient flux:
+  $$
+  \mathbf{H}_s(\mathbf{Q}_h^+, \mathbf{Q}_h^- ; \mathbf{n}) = \frac{1}{2}\left(
+  \mathbf{Q}_h^+ + \mathbf{Q}_h^-\right)\mathbf{n}
+  $$
+
+
+Domain boundary considerations
+------------------------------
+
+What happens when $\partial E \cap \partial\Omega \neq \emptyset$?
+
+In DG, numerical fluxes are not only responsible for handling the flow of information
+between adjacent cells, but they also enforce information at the boundaries.
+
+We denote the *boundary fluxes* as $\mathbf{h}^*_e(\mathbf{Q}_{bc})$,
+$\mathbf{h}^*_v(\mathbf{Q}_{bc}$, $\mathbf{\Sigma}_{bc})$, and
+$\mathbf{H}^*_s(\mathbf{Q}_{bc})$, where $\mathbf{Q}_{bc}$, $\mathbf{\Sigma}_{bc}$ denote
+boundary conditions imposed on the state, and the gradient of the state respectively.
+
+For all $\partial E \cap \partial\Omega$ there is no $+$ side to consider; just the
+interior state ($-$ side) and the prescribed boundary conditions $\mathbf{Q}_{bc},
+\mathbf{\Sigma}_{bc}$.
+
+* At solid walls:
```

Comment:
```suggestion
Solid walls
^^^^^^^^^^^
```
(I prefer that over the bullets, also below)

Reply:
(fa4f487)